### PR TITLE
docs: update tag taxonomy for blog posts

### PR DESCRIPTION
**Pull request by noelweichbrodt**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [X] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [X] I have run `npm test` against my changes and tests pass.
- [X] I have added or edited necessary documentation, or no docs changes are needed.

### Description

Revise the taxonomy used for blog posts to make relating things easier for different content audiences

### CODE REVIEW

File: content/blog/how-to-make-a-better-voice-assistant/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: How to Make a Better Voice Assistant
 date: '2020-08-20'
 description: Google is always trying to improve Google Assistant, but can your app do better?
-tags: Business, Product
+tags: Voice Assistant, Privacy, Doctrine
```
Comment (space-pope) [inline content/blog/how-to-make-a-better-voice-assistant/index.md]: Doctrine :). Might be a bit over the top...maybe "principles", "mission", or even "philosophy"?

File: content/blog/jovo-support-for-spokestack-enables-mobile-app-voice-assistants/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Jovo Support for Spokestack Enables Mobile App Voice Assistants
 date: '2020-07-15'
 description: It's now easier than ever to build your own Independent Voice Assistant. This new integration helps developers build cutting edge conversational experiences in mobile apps.
-tags: Product
+tags: Integration, News, NLU, Dialogue Management
```
Comment (space-pope) [inline content/blog/jovo-support-for-spokestack-enables-mobile-app-voice-assistants/index.md]: "Integration" clashes a bit with the "3rd Party Integration" added to the App Actions posts.

File: content/blog/porting-a-smart-speaker-voice-app-to-mobile-part-1/index.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Porting a Smart Speaker Voice App to Mobile - Part 1
 date: '2020-06-08T00:00:00.000Z'
 description: Spokestack introduces a new way to build and access voice apps independent from major virtual assistant platforms. Take your smart speaker app mobile.
-tags: Engineering, Product
+tags: Engineering, Voice Assistant, 3rd Party Integration, NLU, ASR, Dialogue Management, TTS, Wakeword
```
Comment (space-pope) [inline content/blog/porting-a-smart-speaker-voice-app-to-mobile-part-1/index.md]: This is the first I'm seeing NLU, ASR, and TTS tags—most of our tutorials involve at least a subset of these.

File: content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Porting the Alexa Minecraft Skill to iOS Using Spokestack
 date: '2020-06-08T00:00:04.000Z'
 description: Make a voice-based app for smart speakers. Spokestack makes it easy to convert a smart speaker voice app to a mobile app. Follow our process.
-tags: Engineering, Tutorial
+tags: Engineering, Tutorial, 3rd Party Integration, NLU, iOS, App
```
Comment (space-pope) [inline content/blog/porting-the-alexa-minecraft-skill-to-ios-using-spokestack/index.md]: Also uses ASR and TTS—I could add this comment to most of the other "porting" tutorials.

**Review by space-pope — COMMENTED** [review]
Looks pretty good overall; just had notes on a few of them.

File: content/blog/how-to-make-a-better-voice-assistant/index.md (hunk 1 above)
Comment (noelweichbrodt) [reply content/blog/how-to-make-a-better-voice-assistant/index.md]: Oh, I like `Principles` better. Mostly did Doctrine to get a rise :)

File: content/blog/jovo-support-for-spokestack-enables-mobile-app-voice-assistants/index.md (hunk 2 above)
Comment (noelweichbrodt) [reply content/blog/jovo-support-for-spokestack-enables-mobile-app-voice-assistants/index.md]: Meant to change that, good catch.

**Review by space-pope — APPROVED** [review]

**Pull request merged**